abilities-api.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Plugin Name: Abilities API
+ * Plugin URI: https://github.com/WordPress/abilities-api
+ * Description: Provides a standardized API for registering and executing abilities in WordPress
+ * Version: 0.1.0
+ * Author: WordPress Contributors
+ * License: GPL v2 or later
+ * Text Domain: abilities-api
+ *
+ * @package Abilities_API
+ */
+
+// Prevent direct access
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// TODO: https://github.com/WordPress/abilities-api/pull/4/ also contains this
+
+// Load plugin files
+require_once __DIR__ . '/src/class-wp-ability.php';
+require_once __DIR__ . '/src/class-wp-abilities-registry.php';
+require_once __DIR__ . '/src/abilities-api.php';
+require_once __DIR__ . '/src/rest/class-wp-rest-abilities-init.php';
+
+/**
+ * Initialize the Abilities API.
+ *
+ * @since 0.1.0
+ */
+function abilities_api_init() {
+	/**
+	 * Fires when the Abilities API is ready to register abilities.
+	 *
+	 * @since 0.1.0
+	 */
+	do_action( 'abilities_api_init' );
```

Comment:
Why was it needed here? It should be enough to run the ability registration inside a callback set for the `abilities_api_init` action.